runtime/src/runtime/fvm.rs

```diff
@@ -259,8 +259,6 @@ where
         let state_cid = fvm::sself::root()
             .map_err(|_| actor_error!(illegal_argument; "failed to get actor root state CID"))?;
 
-        log::debug!("getting cid: {}", state_cid);
```

Comment:
We can bring this kind of thing back as `trace`, if needed. But it looks like just noisy remnants of early development.